app.js

```diff
@@ -46,14 +35,7 @@ var configDB = require('./config/database.js');
 // in your own promise library instead: http://mongoosejs.com/docs/promises.html
 mongoose.Promise = global.Promise;
 mongoose.connection.on('error', configDB.check);
-// Connect to proper db
-if (process.env.NODE_ENV == 'production') {
-  mongoose.connect(app.data.mongo_url.prod)
-} else if (process.env.NODE_ENV == 'testing'){
-  mongoose.connect(app.data.mongo_url.test)
-} else {
-  mongoose.connect(app.data.mongo_url.dev)
-}
+mongoose.connect(config.mongodb.link);
```

Comment:
> This was used to run tests locally, on a different db, so it won't affect staging data. It also separate local data from production data.

You shouldn't be able to connect to the remote(production) database from a dev machine. If you want to test the app agains the production database, you should copy it onto your machine. After the tests have finished, we should restore the database to the previous state (we shouldn't modify the local database, even if it's on a dev machine).

Here's what I suggest we should use for testing the backend API.
[https://www.npmjs.com/package/graphql-tester](https://www.npmjs.com/package/graphql-tester)